dist/vl-header.src.js

```diff
@@ -71,6 +76,7 @@ export class VlHeader extends vlElement(HTMLElement) {
       document.body.insertAdjacentHTML('afterbegin', this.getHeaderTemplate());
     }
     eval(code.replace(/document\.write\((.*?)\);/, 'document.getElementById("' + VlHeader.id + '").innerHTML = $1;'));
+    this.dispatchEvent(new CustomEvent(VlHeader.EVENTS.ready));

```

Comment:
ge zijt zeker dat op dees moment dan wel uwen header al een hoogte gekregen heeft en dat uw probleem gaat oplossen?

Reply:
nee.. ik ben daar nu mee aan het worstelen.. bovendien zal de event nog moeten bubbelen denk ik

Reply:
altijd handig dat ge, voor ge releases etc begint te maken gewoon de wijzigingen die ge zo willen zien in uwe node_modules folder ne keer maakt en kijkt of het helpt ...